<launch>
  
  <include file="$(find temoto)/launch/ur3_driver.launch">
    <arg name="robot_ip" value="192.168.1.3" />
  </include>
  
  <include file="$(find temoto_ur3_moveit_config)/launch/temoto_ur3_moveit_planning_execution.launch" />
  
  <include file="$(find temoto_ur3_moveit_config)/launch/moveit_rviz.launch">
    <arg name="config" value="true" />
  </include>
  
  <include file="$(find temoto)/launch/pocketsphinx.launch" />

  <node name="leap_motion" pkg="leap_motion_controller" type="leap_motion" />
    
  <node name="griffin_powermate" pkg="temoto" type="griffin_powermate" />
  <node name="move_robot" pkg="temoto" type="move_robot" args="ur3_stewie" />
  <node name="rviz_visual" pkg="temoto" type="rviz_visual" />
  <node name="interpret_utterance" pkg="temoto" type="interpret_utterance" />
  <node name="human_frame_broadcaster" pkg="temoto" type="human_frame_broadcaster" />

</launch>

